### ISSUE TITLE
wrong name for preload.ts file?

### ISSUE THREAD

**Issue by tmtron** [bennymeg/nx-electron#165]
**Describe the bug**
13.0.0 generates: `dist/apps/app/main.preload.ts.js`  
I think it should not include `.ts`: e.g. `dist/apps/app/main.preload.js`  
because this is also what is used in `app.ts`; `preload: join(__dirname, 'main.preload.js'),` 

**To Reproduce**
1. `npx create-nx-workspace@13 --name=nxelectron13 --preset=angular --appName=ng --style=scss --nxCloud=false`
1. `cd nxelectron13/`
1. `npm install -D nx-electron`
1. `nx g nx-electron:app app --frontendProject=ng --directory=`
1. `nx build ng`
1. `nx build app` 
1. then check the file in `dist/apps/app/`

**Expected behavior**
the file should be called `dist/apps/app/main.preload.js` (without `.ts`)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Nx Electron Version 13.0.0

**Comment by bennymeg** [comment]
Probably fixed in #164.

**Comment by tmtron** [comment]
Is there a workaround until a new version is released? Or is there a simple way to test the current master branch of this repository in my project?

**Comment by paustint** [comment]
@tmtron here is my workaround - add this to your nx project.json config - this will have nx compile and output this file separately from the main app. 

* update your path accordingly
* the entryName will be the output file, e.x. `main.preload.js`

```
        "additionalEntryPoints": [
          {
            "entryName": "main.preload",
            "entryPath": "apps/app-name/src/app/api/main.preload.ts"
          }
```

**Comment by bennymeg** [comment]
Please try v13.1.0

**Comment by tmtron** [comment]
verified, that this is fixed in version 13.1.1
